src/plugins/widgets/index.ts

```diff
@@ -23,6 +24,7 @@ export const widgetConfigs = [
   time,
   todo,
   weather,
+  topSites,
```

Comment:
This should be moved inside an "extension only" check, the inverse of what the `js` widget has.

```ts
if (BUILD_TARGET !== 'web') {
  widgetConfigs.push(topSites);
}
```